### DISCUSSION TITLE
support comments (`jsonc`) in app manifests

### DISCUSSION THREAD

**Discussion by goyalyashpal** [ScoopInstaller/Scoop#5913] [Ideas]
as comments help a lot:
* say in commenting out the details for the architectures u don't use in local manifests
as this will help reduce unnecessary package downloads for updating hashes of those
* for adding blank keys while creating new manifests by hand
* to add any upcoming changes in the app manifest - but while it is not landed in upstream stable yet
* to add any detail worth paying attention to
* etc etc.

**Comment by rashil2000** [comment]
Please update this thread: https://github.com/ScoopInstaller/Scoop/issues/2715

**Reply by goyalyashpal** [reply]
i had read that before, but jsonc is not entirely different thing. it's just json with comments no?
